src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-FSharp/Controllers/WeatherController.fs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Company.WebApplication1.Controllers
```

Comment:
This template explicitly compiles each of its items, so [the fsproj.in](https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/WebApi-FSharp.fsproj.in) is still pointed at the now-missing ValuesController.fs.